src/timepicker/timepicker.utils.ts

```diff
@@ -39,7 +38,7 @@ export function toNumber(value: string | number): number {
     return value;
   }
 
-  return parseInt(value, dex);
+  return +value;
```

Comment:
I suggest to use `Number(value)` which returns `NaN` if it can not be parsed (which I think is the same as the `+value`) but adds a lot more verbosity.